yarn.lock

```diff
@@ -2473,9 +2473,11 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/catalog-model@^0.2.0":
-  version "0.7.0"
+  version "0.2.0"
```

Comment:
oof, this is some old stuff - you probably want to remove the yarn.lock changes entirely, then rebase on a fresh master, then run yarn install again